doc/content/modules/installation-guide/pages/how-tos/install.adoc

```diff
@@ -3,17 +3,41 @@
 This section delves into the detailed step-by-step instructions for installing {product}.
 Detailed explanations walk you through each step.
 
+== Full docker installation of {product}
+
 [INFO]
 ====
 Before you proceed with the installation, please ensure you have the following xref:requirements.adoc[prerequisites]:
 
-* xref:requirements.adoc[Java supported version]
-* https://www.docker.com/[Docker] installed, or an existing xref:requirements.adoc[supported database] installation with a user that has administrator privileges.
+* https://www.docker.com/[Docker] installed.
 ====
 
-== Get {product}
+{product} is distributed as a docker container.
+To install {product}, just download this https://github.com/eclipse-syson/syson/blob/{syson_tag}/docker-compose.yml[docker-compose] file and run the following docker command near the _docker-compose.yml_ file you've just downloaded:
+[source, bash]
+----
+docker compose up
+----
+
+It will install {product} (in the version corresponding to the tagged version of the documentation you are reading) and the postgres database in a composed container. 
+
+If you want to install a specific version, just edit the _docker-compose.yml_ and replace the tagged version by the one you want.
+
+You can also change the version of the documentation (use the menu on the bottom left corner) and download the _docker-compose.yml_ of this version.
+
+All accessible versions of {product} as docker containers are listed https://hub.docker.com/r/eclipsesyson/syson/tags[here].
+
+== Manual installation of {product}
 [#download]
 
+[INFO]
+====
+Before you proceed with the installation, please ensure you have the following xref:requirements.adoc[prerequisites]:
+
+* xref:requirements.adoc[Java supported version]
+* https://www.docker.com/[Docker] installed, or an existing xref:requirements.adoc[supported database] installation with a user that has administrator privileges.
+====
+
 {product} is distributed as a single _executable Java Archive_ `JAR` which contains the complete {product} application along with all its dependencies.
 This distribution simplifies the setup process, enabling convenient exploration of {product}'s features.
 
```

Comment:
I would expect the sections Start the database instance and Start the application would become subsections of the  Installing SysON Manually section.

Reply:
Done